scripts/rollup.config.js

```diff
@@ -4,6 +4,13 @@ const replace = require('./rollup-replace');
 // see https://github.com/videojs/videojs-generate-rollup-config
 // for options
 const options = {
+  globals(defaults) {
+    Object.keys(defaults).forEach(function(type) {
```

Comment:
We shouldn't need this, `Omniture` should be passed in as an option to `videojs-vr`